Plugins/Flow.Launcher.Plugin.Sys/Main.cs

```diff
@@ -77,9 +82,48 @@ public List<Result> Query(Query query)
             return results;
         }
 
+        private string GetDynamicTitle(Query query, Result result)
+        {
+            var pair = KeywordTitleMappings
+                .Where(kvp => kvp.Key == result.Title && kvp.Key != kvp.Value)
+                .FirstOrDefault();
+
+            if (pair.Equals(default))
+            {
+                Log.Error($"Dynamic Title not found for: {result.Title}");
+                return "Title Not Found";
+            }
+
+            var englishTitleMatch = StringMatcher.FuzzySearch(query.Search, pair.Key);
+            var translatedTitleMatch = StringMatcher.FuzzySearch(query.Search, pair.Value);
+
+            return englishTitleMatch.Score >= translatedTitleMatch.Score ? pair.Key : pair.Value;
+        }
+
         public void Init(PluginInitContext context)
         {
             this.context = context;
+            KeywordTitleMappings = new Dictionary<string, string>{
```

Comment:
Probably we don't want to fix translation as user are free to change language when using (though unlikely). I suppose you can have a dictionary mapping to the key of the translation?

Reply:
Sure, that would make sense. I thought that the value from the dictionary would be automatically updated, but you're right. Thanks for the hint!